src/sidebar/store/modules/test/groups-test.js

```diff
@@ -108,4 +139,140 @@ describe('sidebar.store.modules.groups', () => {
       assert.equal(store.focusedGroupId(), privateGroup.id);
     });
   });
+
+  describe('getFeaturedGroups', () => {
+    [
+      {
+        description:
+          'If logged in and there is a restricted group of which the user is not a member show it in `Featured Groups`.',
```

Comment:
This description is a little hard to human-parse...I grokked it in the end. At the least, a comma after "member" would be helpful.